packages/cli/test/build.test.js

```diff
@@ -0,0 +1,42 @@
+import fs from 'node:fs';
+import {execa} from 'execa';
+import {describe, expect, test} from 'vitest';
+
+const cmd = '../../../bin/cli.js';
+
+describe('co build', () => {
+  test('default', async () => {
```

Comment:
new build test that gives some assurance this PR didn’t totally break everything